api/v1/schemas/user.py

```diff
@@ -16,15 +17,23 @@ class UserCreate(UserBase):
 class UserCreateResponse(BaseModel):
     id: str
     username: str
-    email: str
+    email: EmailStr
     access_token: str
     expiry: datetime
 
 
 class UserLogin(BaseModel):
     email: EmailStr
     password: str
-    
+
+
+# Used for comment to provide extra details about user
+class UserResponse(BaseModel):
+    model_config = ConfigDict(from_attributes=True)
+
+    id: str
+    username: str
+    profile_pictures: ProfilePictureResponse | None = None
```

Comment:
Only return the latest profile picture since that's what's going to be rendered on the comment screen. All profile pictures can only be returned on the user profile screen or when necessary.